e2e/cypress/integration/dashboard-transloadit.spec.ts

```diff
@@ -12,6 +12,7 @@ describe('Dashboard with Transloadit', () => {
 
     cy.wait('@assemblies')
     cy.wait('@resumable')
+    cy.wait('@assemblies')
```

Comment:
why 2 wait for @assemblies?

Reply:
There are in fact 5 requests made to assemblies, each wait only waits for one request. Now I could do this five times, but two seems to be plenty for the test to complete.

Reply:
Maybe 5 is the correct thing to do then, if 5 will guarantee that it succeeds? while 1 or 2 will only make it progressively less likely to fail, or am i misunderstanding

Reply:
The thing what makes that unpredictable to me is how many requests will be send by uppy for every file? Is it an interval? Perhaps it can sometimes be 4 instead of 5 because we happy to be lucky with the upload. If we truly know (I don't) how many requests it will send, I could change it.

Reply:
ah. could we instead wait for the ui to change? maybe wait for the success text to show up?

Reply:
Alternatively I use [`contains`](https://docs.cypress.io/api/commands/contains#Syntax) with a high timeout.

Reply:
Normally it is best practice to wait for network request in such circumstances, but because we send an unknown amount of requests I agree.